db_stress_tool/db_stress_env_wrapper.h

```diff
@@ -28,7 +28,9 @@ class DbStressEnvWrapper : public EnvWrapper {
         f.find(".restore") != std::string::npos) {
       return target()->DeleteFile(f);
     }
-    return Status::OK();
+    // Rename the file instead of deletion to keep the history, and
+    // at the same time it is not visible to RocksDB.
+    return target()->RenameFile(f, f + "_renamed_");
```

Comment:
Could there be a risk of db_stress false alarms due to running out of space?

Reply:
Before the change, it didn't delete the file, just returning OK, so from space's point of the view, it should be the same. Manifest is in generally small, so we didn't see out of space so far.